pkg/render/compliance_test.go

```diff
@@ -90,9 +91,90 @@ var _ = Describe("compliance rendering tests", func() {
 		})
 	})
 
+	Context("Management cluster", func() {
+		It("should render all resources for a default configuration", func() {
+			component, err := render.Compliance(nil, &corev1.Secret{
```

Comment:
With the secret specified we should check that the mount is added to the deployment.